### ISSUE TITLE
remote notebook support?

### ISSUE THREAD

**Issue by nbara** [lgeiger/hydrogen-launcher#10]
Hi, I am using hydrogen to connect on a remote jupyter notebook. 

Unless I am mistaken, the launcher doesn't support this (or at least it doesn't work for me). Is that something that could be addressed in the long term?

**Comment by lgeiger** [comment]
> Unless I am mistaken, the launcher doesn't support this

That's correct. At the moment Hydrogen Launcher only supports local kernels and connects to them via their connection file.

I'm sure it's possible to support remote kernels in the future. Feel free to make a PR or ask more questions if like to dig into this.

**Comment by mcr-ksh** [comment]
this would be great

**Comment by TinasheMTapera** [comment]
Just checking in on this, launcher for remote kernel would be really tie together my workflow! If I'm not mistaken, it would be a matter of pulling out the connection file from wherever the remote kernel is running. Not sure how to do this though.

Great product everyone!

**Comment by kylebarron** [comment]
This already exists, without the need to deal with connection files. See https://nteract.gitbooks.io/hydrogen/docs/Usage/RemoteKernelConnection.html.

**Comment by kylebarron** [comment]
Oh my bad, I thought this was the main hydrogen repo

**Comment by jwdejong** [comment]
Hi what would be necessary to support this? It would be really nice to have the possibility of running a ipython console with docker / remote interpreter.
